test/useTranslation.loading.spec.js

```diff
@@ -49,73 +15,32 @@ describe('useTranslation loading ns', () => {
     newI18n.use(backend).init({
       lng: 'en',
       fallbackLng: 'en',
-
       interpolation: {
         escapeValue: false, // not needed for react!!
       },
     });
   });
 
-  const TestElement = ({ useSuspense = true }) => {
-    const [t] = useTranslation('common', { i18n: newI18n, useSuspense });
-    return <div>{t('key1')}</div>;
-  };
-
-  it('should wait for correct translation', done => {
-    const spy = sinon.spy();
-
-    console.error = jest.fn(); // silent down the error boundary error from react-dom
-    let wrapper = mount(
-      <ErrorBoundary spy={spy}>
-        <TestElement />
-      </ErrorBoundary>,
+  it('should wait for correct translation RTL', async () => {
+    const { result, waitForNextUpdate } = renderHook(() =>
+      useTranslation('common', { i18n: newI18n, useSuspense: true }),
     );
-
-    expect(wrapper.contains(<div>Something went wrong.</div>)).toBe(true);
-    expect(spy.callCount).toBe(1);
-
+    expect(result.current).toBe(null);
     backend.flush();
-
-    setTimeout(() => {
-      // mount again - no suspense recovering from promise resolve
-      wrapper = mount(
-        <ErrorBoundary spy={spy}>
-          <TestElement />
-        </ErrorBoundary>,
-      );
-
-      // console.log(wrapper.debug());
-      expect(wrapper.contains(<div>test</div>)).toBe(true);
-      done();
-    }, 500);
+    await waitForNextUpdate();
+    const { t } = result.current;
+    expect(t('key1')).toBe('test');
   });
 
-  it('should wait for correct translation without suspense', done => {
-    const spy = sinon.spy();
-
-    let wrapper = mount(
-      <ErrorBoundary spy={spy}>
-        <TestElement useSuspense={false} />
-      </ErrorBoundary>,
+  it('should wait for correct translation RTL', async () => {
```

Comment:
shouldn't this be `'should wait for correct translation without suspense'`?

Reply:
You are right! :raised_hands: